converters/google/convert.go

```diff
@@ -240,21 +250,21 @@ func (c *Converter) addDelete(rc *tfjson.ResourceChange) error {
 			} else if !c.offline {
 				asset, err := converter.FetchFullResource(&rd, c.cfg)
 				if errors.Cause(err) == resources.ErrEmptyIdentityField {
-					c.errorLogger.Warn(fmt.Sprintf("%s did not return a value for ID field. Skipping asset fetch.", key))
+					c.errorLogger.Debug(fmt.Sprintf("%s: Unable to fetch and merge remote %s asset due to unset or (known after apply) identity fields on the TF resource.", rc.Address, converted.Type))
 					existingConverterAsset = nil
 				} else if errors.Cause(err) == resources.ErrResourceInaccessible {
-					c.errorLogger.Warn(fmt.Sprintf("%s was not able to be fetched due to not existing or insufficient permission. Skipping asset fetch.", key))
+					c.errorLogger.Warn(fmt.Sprintf("%s: Fetching %s for merge failed due to not existing or insufficient permission.", rc.Address, key))
 					existingConverterAsset = nil
 				} else if err != nil {
-					return errors.Wrap(err, "fetching asset")
+					return fmt.Errorf("fetching remote asset %s: %w", key, err)
```

Comment:
It seems intentional for a few of the conditional blocks you have, but I'm curious why we are not prefixing these errors with an `rc.Address` like the others